.github/workflows/release-chart.yml

```diff
@@ -36,3 +37,21 @@ jobs:
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
           CR_GENERATE_RELEASE_NOTES: true
+
+      - name: Login to GHCR
+        uses: docker/login-action@v3.4.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push Helm chart to GHCR as OCI artifact
+        run: |
+          chart_registry="ghcr.io/${GITHUB_REPOSITORY_OWNER}/charts"
```

Comment:
@Nerkho @JanMa are we okay with `ghcr.io/openbao/charts`  ?

Reply:
I don't mind starting with this, but we should look into moving it to quay.io afterward as the container images.